frontend/src/reviews/EditableReviewComment.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, Stack, Typography } from '@mui/material'
+import MarkdownDisplay from 'src/common/MarkdownDisplay'
+import RichTextEditor from 'src/common/RichTextEditor'
+import MessageAlert from 'src/MessageAlert'
+import { ResponseInterface } from 'types/types'
+import { formatDateTimeString } from 'utils/dateUtils'
+
+interface EditableReviewCommentProps {
+  comment: string
+  setComment: (value: string) => void
```

Comment:
`set...` should be reserved for react's set state functions. This is just a callback prop that happens to lead to a set state call, but this component should have no knowledge of how the value is to be used. Rename to `onCommentChange`?